src/reaction.rs

```diff
@@ -223,7 +236,15 @@ pub async fn send_reaction(context: &Context, msg_id: MsgId, reaction: &str) ->
     let reaction_msg_id = send_msg(context, chat_id, &mut reaction_msg).await?;
 
     // Only set reaction if we successfully sent the message.
-    set_msg_id_reaction(context, msg_id, msg.chat_id, ContactId::SELF, reaction).await?;
+    set_msg_id_reaction(
+        context,
+        msg_id,
+        msg.chat_id,
+        ContactId::SELF,
+        reaction_msg.get_timestamp(),
```

Comment:
I'd pass `timestamp_sort` explicitly. `timestamp_sent` is 0 for outgoing messages anyway and also you pass `timestamp_sort` for incoming messages in `receive_imf.rs`

Reply:
i added a commit to change that.